plugins/inputs/twitter/twitter.go

```diff
@@ -0,0 +1,100 @@
+package twitter
+
+import (
+	"net/url"
+	"strings"
+
+	"github.com/ChimeraCoder/anaconda"
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/plugins/inputs"
+)
+
+const (
+	measurement = "tweets"
+)
+
+// Twitter - The object used to hold the configuration options from the user.
+type Twitter struct {
+	ConsumerKey       string
+	ConsumerSecret    string
+	AccessToken       string
+	AccessTokenSecret string
+	KeywordsToTrack   string
+}
+
+// Description - returns a description of the plugin
+func (s *Twitter) Description() string {
+	return "a plugin for pulling metrics about tweets"
+}
+
+// SampleConfig - returns a sample config block for the plugin
+func (s *Twitter) SampleConfig() string {
+	return `
+    ## These values can be obtained from apps.twitter.com
+    consumerKey = ""
+    consumerSecret = ""
+    accessToken = ""
+    accessTokenSecret = ""
+    keywordsToTrack = ""
+  `
+}
+
+// Gather - Called every interval and used for polling inputs
+func (s *Twitter) Gather(acc telegraf.Accumulator) error {
+	return nil
+}
+
+// Start - Called once when starting the plugin
+func (s *Twitter) Start(acc telegraf.Accumulator) error {
+	anaconda.SetConsumerKey(s.ConsumerKey)
+	anaconda.SetConsumerSecret(s.ConsumerSecret)
+	api := anaconda.NewTwitterApi(s.AccessToken, s.AccessTokenSecret)
+	go s.fetchTweets(api, acc)
+	return nil
+}
+
+// Stop - Called once when stopping the plugin
+func (s *Twitter) Stop() {
+	return
```

Comment:
Needs to stop goroutine started in Start, or things like reloading via SIGHUP will have problems.